src/runtime_src/core/pcie/tools/cloud-daemon/azure/azure.cpp

```diff
@@ -233,6 +249,13 @@ int AzureDev::azureLoadXclBin(const xclBin *buffer)
     if (fpgaSerialNumber.empty())
         return -E_EMPTY_SN;
     std::cout << "LoadXclBin FPGA serial No: " << fpgaSerialNumber << std::endl;
+
+    // check if the xclbin is valid
+    if (xclbin_section_hdr_present(buffer, BITSTREAM)) {
```

Comment:
We dont need new function. Please use existing function to check the existence of section